src/openMVG/sfm/pipelines/sequential/sequential_SfM.cpp

```diff
@@ -1416,7 +1414,12 @@ bool SequentialSfMReconstructionEngine::Resection(const std::size_t viewIndex)
         const Vec2 xJ = _features_provider->feats_per_view.at(J)[track.at(J)].coords().cast<double>();
 
         // test if the track already exists in 3D
-        if (_sfm_data.structure.count(trackId) != 0)
+        bool trackIdExists;
+#pragma omp critical
```

Comment:
@fabiencastan 
Also, unrelated to this issue, I wonder whether it would be better to triangulate points running through the visible tracks instead of the views: instead of trying to triangulate points for each couple of views, we should try to get the common tracks between the current `viewIndex` and all the rest, and try to triangulate or update the observations. 
And the same, if we resection a group of images, it would be better to use all the available observations to reconstruct the points instead of relying only on 2-view reconstruction. But ok this is a matter of other issue